docs/room/realtimemodel-primitives.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs
+title: RealtimeModel Primitives
+---
+# RealtimeModel Primitives
+
+In [RealtimeModel](./realtimemodel.md) definitions the `[RealtimeProperty]` attribute can be used on properties of many different types. Here's the full list:
+
+# C# primitives
+* `bool`
+* `byte`
+* `sbyte`
+* `short`
+* `ushort`
+* `int`
+* `uint`
+* `float`
+* `double`
+* `string`
+
+# Unity primitives
+* `Color`
+* `Vector2`
+* `Vector3`
+* `Vector4`
+* `Quaternion`
+
+# Collections
+* `RealtimeArray`
+* `RealtimeSet`
+* `RealtimeDictionary`
+* `StringKeyDictionary`
+
+Our collection classes implement change tracking to create efficient network updates and to resolve conflicts between clients.
```

Comment:
I don't think this comment is explicit enough. It's hard to tell what kind of change tracking the collections do and it's not safe to assume the collections are checking the model primitive values. I think it's worth referencing the Collections page copy or sending folks to that page instead of adding info about collections here.